internal/services/recoveryservices/site_recovery_replicated_vm_resource_test.go

```diff
@@ -757,6 +990,74 @@ resource "azurerm_site_recovery_replicated_vm" "test" {
 `, r.template(data), data.RandomInteger)
 }
 
+func (r SiteRecoveryReplicatedVmResource) withFivePointOhTFOSettings(data acceptance.TestData) string {
```

Comment:
We should rename the deprecated test config rather than including a version for this new test config, e.g. `withDeprecatedTFOSettings`

```suggestion
func (r SiteRecoveryReplicatedVmResource) withTFOSettings(data acceptance.TestData) string {
```